Options.lua

```diff
@@ -9,32 +9,134 @@ local L = LibStub("AceLocale-3.0"):GetLocale("TeleportMenu")
 -------------------------------------
 -- Locales
 --------------------------------------
-
-local defaultsDB = {
-    enabled = true,
-    iconSize = 40,
-    hearthstone = "none",
-    maxFlyoutIcons = 5,
-    reverseMageFlyouts = false,
-    buttonText = true,
-    showOnlySeasonalHerosPath = false
+SettingsBase = {
+	__index = {
+		["Enabled"] = false,
```

Comment:
I didn't see a conversion function that will relocate old -> new settings variables, don't want to reset settings for people when changing this.